functests/4_latency/latency.go

```diff
@@ -255,6 +228,65 @@ var _ = Describe("[performance] Latency Test", func() {
 	})
 })
 
+//read LATENCY_TEST_RUN environment var and parse it if set
+func getLatencyTestRun() {
+	latencyTestRunEnv := os.Getenv("LATENCY_TEST_RUN")
+	if latencyTestRunEnv != "" {
+		if latencyTestRunEnv == "true" {
+			latencyTestRun = true
+		}
+	}
+}
+
+//read LATENCY_TEST_RUNTIME environment var and parse it if set
+func getLatencyTestRuntime() {
+	latencyTestRuntimeEnv := os.Getenv("LATENCY_TEST_RUNTIME")
+	if latencyTestRuntimeEnv != "" {
+		latencyTestRuntime = latencyTestRuntimeEnv
```

Comment:
This is still caching. The getter should just parse the value and return the string. The `latencyTestRuntime` variable should be removed

Reply:
@MarSik should we still check the condition if the value is not empty?

Reply:
because the intention in the initial code was to set new value into the vars if the new value is not empty , otherwise to keep it default

Reply:
ti seems a good opportunity to use `LookupEnv` + `ParseBool`:
```
func getLatencyTestRun() {
        // TODO: log if/where needed
        if envValue, ok := os.LookupEnv("LATENCY_TEST_RUN"); ok {
            if value, err := strconv.ParseBool(envValue); err == nil {
                 return value // this could be true or false, but it explicitely signal the intent of the user
            }    
        }
        // something was wrong or missing, stick with the builtin default
      return false
}
```

Reply:
yes good one, I used it. thanks .